tests/storage/restore.go

```diff
@@ -1681,9 +1681,7 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 					memoryDumpPVC.Spec.VolumeMode = &volumeMode
 					var err error
 					memoryDumpPVC, err = virtClient.CoreV1().PersistentVolumeClaims(testsuite.GetTestNamespace(nil)).Create(context.Background(), memoryDumpPVC, metav1.CreateOptions{})
-					if err != nil {
```

Comment:
@ShellyKa13 does this make sense? I was pretty confused by it (PVC will get created, just won't bind)

Reply:
fair point, will do

Reply:
actually, are you sure it's needed?
it's already labelling & grabbing the snapshot storage class, and I am not that worried about it not being able to provision filesystem PVCs

Reply:
yeah I get that its not probable that there isnt a filesystem storage, but I want to prevent failure of the test due to memory dump in a restore test

Reply:
I would have to write code that checks snapshotStorageClass is filesystem capable
(tbh, there is no way to know other than trying to create a PVC)

I'm not sure it's worth it - those tests already Fail() if there's no snapshot class.
If it exists, it can do filesystem storage
(same point I was making in the first commit about fs rwo being consensus)